logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/creating/basic/contains/creators/impl/ContainsObjectsAssertionCreator.kt

```diff
@@ -32,17 +32,17 @@ abstract class ContainsObjectsAssertionCreator<T : Any, TT : Any, in SC, S : Con
     checkers: List<C>
 ) : ContainsAssertionCreator<T, TT, SC, C>(searchBehaviour, checkers) {
 
-    final override fun searchAndCreateAssertion(
+    override fun searchAndCreateAssertion(
```

Comment:
I don't know if this is ideal, but I overrode `searchAndCreateAssertion` in the `InAnyOrderValuesAssertionCreator` to handle the case we're changing.

Reply:
Maybe not but I wonder, what was the reason that you did it? was decorateInAnyOrderAssertion not good enough?